extensions/amp-analytics/0.1/events.js

```diff
@@ -410,6 +525,149 @@ export class IniLoadTracker extends EventTracker {
 }
 
 
+/**
+ * Timer event handler.
+ */
+class TimerEventHandler {
```

Comment:
I generally prefer that the `events.js` is more of an assembling place for trackers and any substantial other classes go into a separate file. But this is up to @zhouyx

Reply:
These are very closely coupled to the point where if this were another language I might make this a private class of the other. I would prefer to keep this unexportable, basically.

Reply:
I am OK with this right now. I would prefer to move every EventHandler to their own file if not for circular dependency issue : ( 
I can always refactor once I feel the file is getting too long : )